molo/yourwords/admin.py

```diff
@@ -0,0 +1,18 @@
+from molo.core.models import ArticlePage
+from molo.yourwords.models import YourWordsCompetitionEntry
+from django.contrib import admin
+
+
+def ConvertToArticle(model_admin, request, entry):
+    article = ArticlePage.objects.create(null, null)
```

Comment:
So...not quite how creating pages works in Wagtail..There's a tree structure and all pages need to be added in a tree. Have a look at [these tests](https://github.com/praekelt/molo/blob/develop/molo/core/tests/test_models.py#L52-L71) for an example. According to the SoW, the new article will need to be added to a Section called "True Story" or something.. Let's chat in slack